package.json

```diff
@@ -115,7 +115,7 @@
     "@types/react-bootstrap-table-next": "^4.0.10",
     "@types/react-dom": "^16.9.10",
     "@types/react-router-dom": "^5.1.7",
-    "axios": "^0.21.1",
+    "axios": "^0.21.4",
```

Comment:
Don't update versions unless justified / required as it could cause mysterious problems